CMakeLists.txt

```diff
@@ -126,7 +130,17 @@ if (NOT ENABLE_NET_RECEIVE_BUFFERING)
     add_definitions("-DNET_RECEIVE_BUFFERING=0")
 endif()
 
-find_package(mod2c 2.0.0  REQUIRED)
+set(ExternalProjectCMakeArgs
+     -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+     -DCMAKE_INSTALL_PREFIX=${PROJECT_BINARY_DIR}/external/mod2c
+     -DCMAKE_C_COMPILER=${FRONTEND_C_COMPILER} -DCMAKE_CXX_COMPILER=${FRONTEND_CXX_COMPILER}
+   )
+ExternalProject_Add(mod2c
+  UPDATE_COMMAND ""
+  SOURCE_DIR ${PROJECT_SOURCE_DIR}/external/mod2c
+  INSTALL_DIR ${PROJECT_BINARY_DIR}/external
+  CMAKE_ARGS ${ExternalProjectCMakeArgs}
+  )
```

Comment:
Often we do `rm -rf build`. Hence I am thinking mod2c source should be cloned outside build?

Reply:
During development I am modifying mod2c and accidentally do `rm -rf *`. Oops...!